plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/ConfigureDependentFrameworksListener.java

```diff
@@ -221,14 +217,13 @@ private ADBInfo loadADBInfo(IServer server) {
 				}
 			}
 			return new ADBInfo(adbEnv);
-		} catch(IOException ce) {
-			ce.printStackTrace(); // TODO log
+		} catch(IOException | TimeoutException ce) {
+			CDKCoreActivator.pluginLog().logError("Unable to successfully complete a call to vagrant adbinfo. ", ce);
 		}
 		return null;
 	}
 	
 	private static class ADBInfo {
-		// Mocked
 		private int openshiftPort = 8443;
```

Comment:
What are the benefits to making the inner static class final? I don't imagine there are many here. Since it's private, no subclass can override it anyway.